pysph/sph/gas_dynamics/psph/scheme.py

```diff
@@ -0,0 +1,273 @@
+
+        Tip: Reduce the number of points if particle penetration is
+        encountered. This has to be done while running
+        ``gas_dynamics.wc_blastwave`` and ``gas_dynamics.robert``.
+
+        Parameters
+        ----------
+        fluids: list
+            List of names of fluid particle arrays.
+        solids: list
+            List of names of solid particle arrays (or boundaries), currently
+            not supported
+        dim: int
+            Dimensionality of the problem.
+        gamma: float
+            :math:`\\gamma` for Equation of state.
+        hfact: float
+            :math:`h_{fact}` for smoothing length adaptivity, also referred to
+            as kernel_factor in other gas dynamics schemes.
+        betab : float, optional
+            :math:`\\beta_b` for artificial viscosity, by default 2.0
+        fkern : float, optional
+            :math:`f_{kern}`, Factor to scale smoothing length for equivalence
+            with classic kernel when using kernel with altered
+            `radius_scale` is being used, by default 1.
+        max_density_iterations : int, optional
+            Maximum number of iterations to run for one density step,
+            by default 250.
+        alphac : float, optional
+            :math:`\\alpha_c` for artificial conductivity, by default 0.25
+        density_iteration_tolerance : float, optional
+            Maximum difference allowed in two successive density iterations,
+            by default 1e-3
+        has_ghosts : bool, optional
+            if ghost particles (either mirror or periodic) is used, by default
+            False
+        alphamin : float, optional
+            :math:`\\alpha_{min}` for artificial viscosity switch, by default
+            0.02
+        alphamax : float, optional
+            :math:`\\alpha_{max}` for artificial viscosity switch, by default
+            2.0
+        betac : float, optional
+            :math:`\\beta_c` for artificial viscosity switch, by default 0.7
+        betad : float, optional
+            :math:`\\beta_d` for artificial viscosity switch, by default 0.05
+        betaxi : float, optional
+            :math:`\\beta_{\\xi}` for artificial viscosity switch,
+            by default 1.0
+        """
+
+        self.fluids = fluids
+        self.solids = solids
+        self.dim = dim
+        self.solver = None
+        self.gamma = gamma
+        self.betab = betab
+        self.hfact = hfact
+        self.density_iteration_tolerance = density_iteration_tolerance
+        self.max_density_iterations = max_density_iterations
+        self.has_ghosts = has_ghosts
+        self.fkern = fkern
+        self.alphac = alphac
+        self.alphamin = alphamin
+        self.alphamax = alphamax
+        self.betac = betac
+        self.betad = betad
+        self.betaxi = betaxi
+
+    def add_user_options(self, group):
+        group.add_argument("--alphamax", action="store", type=float,
```

Comment:
`--alpha-max` maybe?  This is a minor nit but just trying to be consistent. Similar for the others here.